### ISSUE TITLE
Resolve failures in MPI tests

### ISSUE THREAD

**Issue by cwschilly** [Pressio/rom-tools-and-workflows#32]
Follow up to #5

Failures can be seen [here](https://github.com/Pressio/rom-tools-and-workflows/actions/runs/6576656816/job/17866551865?pr=18)

**Comment by cwschilly** [comment]
```
 FAILED tests/romtools/test_trial_space_utils/test_svd_kernel_trick_mpi.py::test_parallel_kernel_trick_on_three_cores - AttributeError: module 'romtools.trial_space_utils' has no attribute '__svdMethodOfSnapshotsImpl'
```

**Comment by cwschilly** [comment]
Closed with PR #18
